solutions/usaco-944.mdx

```diff
@@ -196,6 +196,8 @@ for i in range(n):
 
 		ax = bx = coors[i][0]
 		ay = by = coors[i][1]
+		
+		// bfs 
```

Comment:
this is the python solution?

Reply:
you don't use `//` for python comments

Reply:
oh yeah im sorry changing

Reply:
ok but `bfs` tells us literally nothing

Reply:
im guessing changing bfs for breadth first seach would only make u mad.... idk what to put.. that section of the code is basically a direct impl of bfs

Reply:
also what does bump mean

Reply:
"hey this exists, please address it"

Reply:
again, _why_ are you BFS-ing? what are you filling up? what data does this give?

Reply:
> "hey this exists, please address it"

oh... huh i thought it meant get rid of like extra line / space above code... like "bump up" code..

Reply:
> again, _why_ are you BFS-ing? what are you filling up? what data does this give?

okay i will address this now